app/components/repository-sidebar.js

```diff
@@ -68,20 +70,18 @@ export default Ember.Component.extend({
     return runningAmount + queuedAmount;
   },
 
-  @computed('features.proVersion', 'jobState.runningJobs.[]')
-  runningJobs(proVersion, runningJobs) {
-    if (!proVersion) { return []; }
+  @computed('jobState.runningJobs.[]')
+  runningJobs(runningJobs) {
     return runningJobs;
   },
 
-  @computed('features.proVersion')
-  queuedJobs(proVersion) {
-    if (!proVersion) { return []; }
-
+  @computed('currentUser')
+  queuedJobs(user) {
     const queuedStates = ['created'];
     const result = this.get('store').filter(
       'job',
-      job => queuedStates.includes(job.get('state'))
+      job =>
+        queuedStates.includes(job.get('state') && user && user.hasAccessToRepo(job.get('repo')))
```

Comment:
@backspace yes, I think that it's the only easy way to do it.

One interesting thing here, though, is that I realised my diagnosis of a problem is wrong. `store.filter` will actually re-evaluate records when their property change, but it will only happen for defined properties. So for example when you use filter on jobs, this will trigger a re-evaluation of a record: `job.set('state', 'created')`, but this won't: `job.set('foo', 'bar')`. This won't work here, because what you need here is to depend on a property from another object, but I just wanted to clarify what I found.

I plan to merge it soonish, probably today or tomorrow, so I hope that you won't have to wait long.